app/api_routes.js

```diff
@@ -435,7 +414,7 @@ routes.delete(
 routes.delete(
   '/api/v1/users/:user_id/login-token',
   cors(),
-  resources.v1.user_session.delete
+  resources.v1.users.logout
```

Comment:
@whitmanschorn Can you tell me more about why this can't use `user_session.delete`?